doc/tools/docgram/orderedGrammar

```diff
@@ -2339,6 +2339,16 @@ tac2rec_fieldexpr: [
 | qualid ":=" ltac2_expr1      (* ltac2 plugin *)
 ]
 
+tac2rec_fieldpats: [
+| tac2rec_fieldpat ";" OPT tac2rec_fieldpats      (* ltac2 plugin *)
+| tac2rec_fieldpat ";"      (* ltac2 plugin *)
+| tac2rec_fieldpat      (* ltac2 plugin *)
+]
+
+tac2rec_fieldpat: [
+| qualid ":=" tac2pat1      (* ltac2 plugin *)
+]
+
```

Comment:
For readability, please move these lines to just before line 2361 ("ltac2_tactic_atom: [")